src/main/java/net/geforcemods/securitycraft/datagen/BlockTagGenerator.java

```diff
@@ -62,11 +62,7 @@ protected void registerTags()
 				SCContent.REINFORCED_PODZOL.get(),
 				SCContent.REINFORCED_MYCELIUM.get());
 		getBuilder(SCTags.Blocks.REINFORCED_END_STONES).add(
-				SCContent.REINFORCED_END_STONE.get(),
-				SCContent.REINFORCED_END_STONE_BRICKS.get(),
-				SCContent.REINFORCED_END_STONE_BRICK_SLAB.get(),
-				SCContent.REINFORCED_END_STONE_BRICK_STAIRS.get(),
-				SCContent.REINFORCED_END_STONE_BRICK_WALL.get());
+				SCContent.REINFORCED_END_STONE.get());

```

Comment:
When changing things in the datagen package, ALWAYS rerun the `runData` run config.